src/common/pkcs11_lib.c

```diff
@@ -1811,8 +1811,9 @@ int sign_value(pkcs11_handle_t *h, cert_object_t *cert, CK_BYTE *data,
     return -1;
   }
   *signature = NULL;
-  *signature_length = 64;
+  *signature_length = 1024;
```

Comment:
Why do we need a larger initial buffer length here? There is the increment step below:

`*signature_length = current_signature_length * 2;`